Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/TextInput.swift

```diff
@@ -149,7 +108,7 @@ private extension TextInput {
                     }
                 }
             }
-            .scrollContentBackgroundHidden()
+            .scrollContentBackground(.hidden)
```

Comment:
Does [this property](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/main/Sources/ArcGISToolkit/Components/Bookmarks/Bookmarks.swift#L176-L180) need updated too?

Reply:
Good question, this area has is being updated in #549.